pkg/cmd/cli/describe/describer.go

```diff
@@ -230,6 +234,7 @@ func describeBuildSpec(p buildapi.BuildSpec, out *tabwriter.Writer) {
 		formatString(out, "Build Secrets", strings.Join(result, ","))
 	}
 	if len(p.Source.Images) == 1 && len(p.Source.Images[0].Paths) == 1 {
+		noneType = false
```

Comment:
you need to set noneType to false inside the for loop of the else condition (or otherwise set it to false when len(p.Source.Images)>=1, not just ==1)